### ISSUE TITLE
Runtime panic with go 1.6

### ISSUE THREAD

**Issue by laibulle** [ninjasphere/go-openzwave#4]
Not sure if this package is still maintained.

When I use go 1.5, the library seems to work. But with go 1.6 I get this error

```
panic: runtime error: cgo argument has Go pointer to Go pointer

goroutine 8 [running]:
panic(0x40fa8e0, 0xc82000a2e0)
    /usr/local/Cellar/go/1.6/libexec/src/runtime/panic.go:464 +0x3e6
github.com/ninjasphere/go-openzwave._cgoCheckPointer0(0x40d1a00, 0xc8200600c0, 0x0, 0x0, 0x0, 0x0)
    ??:0 +0x4d
github.com/ninjasphere/go-openzwave.(*api).Run.func2(0xc8200600c0, 0xc820056150)
    /Users/laibulle/Eliocity/www/go/src/github.com/ninjasphere/go-openzwave/run.go:113 +0x62
created by github.com/ninjasphere/go-openzwave.(*api).Run
    /Users/laibulle/Eliocity/www/go/src/github.com/ninjasphere/go-openzwave/run.go:198 +0x178
exit status 2
```

**Comment by jonseymour** [comment]
Thanks @laibulle I will have a look at it.

**Comment by jonseymour** [comment]
Seems I have fallen foul of this change - https://github.com/golang/go/issues/12416 .  I will have to re-examine how I share a references to go structures with the C world.

**Comment by jonseymour** [comment]
Closed via #5
